### PR TITLE
docs: keep the same style across documentation & fix some typos

### PR DESCRIPTION
**Pull request by grtcdr**
This MR fixes a few typos I encountered while reading through lualine's documentation. 

I've also tried to reformat ~~parts~~ most of the documentation to keep the same style across the whole thing.

### CODE REVIEW

**Comment by shadmansaleh** [comment]
This looks good . Thanks for taking your time to work on this . We can merge it once the comments are addressed.

File: README.md [hunk 1]
```diff
@@ -348,40 +348,51 @@ sections = {
...
       -- 'highlight_group_name' | {fg = '#rrggbb'|cterm_value(0-255)|'color_name(red)', bg= '#rrggbb', gui='style'}
-      --  Note: all other color options including themes accept  like diff_color same color values
-      -- Example
-      -- color = {fg = '#ffaa88', bg = 'grey', gui='italic,bold'},
-      -- color = {fg = 204} -- when fg/bg is skiped they default to themes fg/bg
-      -- color = 'WarningMsg'
-      -- or highlight group
-      -- color = "WarningMsg"
+      --
+      -- Note: all other color options including themes accept like diff_color same color values.
```
[the remark this answers is not in the document]
Comment (grtcdr) [reply README.md:373]: Yeah, I couldn't make sense of the line initially :/

File: README.md [hunk 2]
```diff
@@ -461,19 +473,19 @@ sections = {
   lualine_a = {
     {
       'diff',
-      colored = true, -- displays diff status in color if set to true
-      -- all colors are in format #rrggbb
+      colored = true, -- Displays diff status in color if set to true
+      -- All colors are in #rrggbb format
```
[the remark this answers is not in the document]
Comment (grtcdr) [reply README.md]: Done :)

**Comment by grtcdr** [comment]
> Thanks for taking your time to work on this

You're welcome. I really like lualine, and I hope to see it succeed for a very very long time.

**Pull request merged**